ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -29,43 +30,57 @@
 '''
 
 LOOKAHEAD_WPS = 50  # Number of waypoints we publish
+MAX_DECEL = 5.
```

Comment:
Please get this from the Parameter Server, so that we don't have duplicates. Should be done in `__init__`, after `rospy.init_node`. For example:

        self.max_acc = rospy.get_param('/dbw_node/accel_limit')